package.json

```diff
@@ -1,48 +1,46 @@
 {
-  "name": "@nuxtjs/router",
-  "version": "1.1.0",
+  "name": "@ax2/router",
```

Comment:
Typo here?

Reply:
Wow sorry about that @pi0! I needed to publish the package under our own scope and I pushed it to the master branch... silly me! Let me redo the PR

Reply:
There you go: https://github.com/nuxt-community/router-module/pull/25